### PR TITLE
return mailing list information about groups

### PR DESCRIPTION
**Pull request by StephenCoady**
bugzilla sync toddler uses the email of a group so this change exposes that

Signed-off-by: Stephen Coady <scoady@redhat.com>

### CODE REVIEW

**Comment by abompard** [comment]
This seems fine. Do you want to add the other group attributes (`fasURL` and `fasIRCChannel`) too or do you prefer it being done in a later PR?

**Comment by StephenCoady** [comment]
I can add them here.

File: fasjson/lib/ldap/models.py [hunk 1]
```diff
@@ -91,6 +91,8 @@ class GroupModel(Model):
         "groupname": Converter("cn"),
         "description": Converter("description"),
         "mailing_list": Converter("fasmailinglist"),
+        "url": Converter("fasurl"),
+        "irc": Converter("fasircchannel"),
```
Comment (abompard) [inline fasjson/lib/ldap/models.py]: It should be multi-valued, thankfully there's a `multivalued` arg ;-)

File: fasjson/tests/unit/test_web_resource_v1_groups.py [hunk 2]
```diff
@@ -18,6 +18,8 @@ def test_groups_success(client, gss_user, mock_ldap_client):
                 "groupname": name,
                 "description": f"the {name} group",
                 "mailing_list": f"{name}@groups.com",
+                "url": f"www.{name}.com",
+                "irc": f"[#{name}]",
```
Comment (abompard) [inline fasjson/tests/unit/test_web_resource_v1_groups.py]: It should be a list, not the string representation of a list.

**Review by abompard — CHANGES_REQUESTED** [review]
Sorry it's a tiny bit more complex than for the mailing-list.

File: fasjson/lib/ldap/models.py (hunk 1 above)
Comment (StephenCoady) [reply fasjson/lib/ldap/models.py]: ah great catch, I forced the tests to pass knowing it should be - but forgot it here.

**Review by abompard — APPROVED** [review]
Thanks!

**Pull request merged**